src/test/java/fr/esgi/al5_2/Tayarim/ProprietaireControllerTest.java

```diff
@@ -18,297 +16,366 @@
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.http.MediaType;
-import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.test.web.servlet.MockMvc;
 
 import fr.esgi.al5_2.Tayarim.services.ProprietaireService;
 import fr.esgi.al5_2.Tayarim.controllers.ProprietaireController;
-import fr.esgi.al5_2.Tayarim.dto.proprietaire.ProprietaireCreationDTO;
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.ProprietaireCreationDto;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'fr.esgi.al5_2.Tayarim.dto.proprietaire.ProprietaireCreationDto' import. Should be before 'org.springframework.test.web.servlet.MockMvc'.